### DISCUSSION TITLE
Agent nginx 443 will report a Hydration completed but contains mismatches.

### DISCUSSION THREAD

**Discussion by spelling203** [nuxt/nuxt#16266] [Questions]
nginx          

         listen              443 ssl http2;
        server_name         new.test.com;

        ssl_certificate     cert/test.com.pem;
        ssl_certificate_key cert/test.com.key;
        ssl_session_timeout 1d;
        ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
        ssl_prefer_server_ciphers   on;
        add_header Strict-Transport-Security "max-age=63072000; includeSubdomains; preload" always;
        add_header X-XSS-Protection "1; mode=block";
        add_header X-Content-Type-Options "nosniff";

        proxy_set_header    Host    $host; 
        proxy_set_header    X-Real-IP   $remote_addr; 
        proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header    X_CLIENT_SOURCE_IP  $http_x-client-source-ip;
        proxy_set_header    User-Agent  $http_user_agent;
        proxy_http_version 1.1;

![image](https://user-images.githubusercontent.com/78407334/204137960-9e9a036b-8c42-4ee6-82f3-392ef510b4aa.png)

But with 127.0.0.1:3000, no error is displayed

Can help me

**Comment by TheAlexLichter** [comment]
This is a classic [hydration error](https://blog.lichter.io/posts/vue-hydration-error/). Sadly it can have lots of reasons 🙈
